content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -30,7 +30,7 @@ it simplifies node deployment and configuration management.
 
 The subset of the Kubelet's configuration that can be configured via a file
 is defined by the `KubeletConfiguration` struct
-[here (v1beta1)](https://github.com/kubernetes/kubernetes/blob/{{< param "docsbranch" >}}/pkg/kubelet/apis/config/types.go).
+[here (v1beta1)](https://github.com/kubernetes/kubernetes/blob/{{< param "docsbranch" >}}/staging/src/k8s.io/kubelet/config/v1beta1/types.go).
```

Comment:
would be good to move this out to a `content/en` PR.

sig-docs-zh team can take care of the Chinese side 😄 

/lgtm

Reply:
ok